solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/List/ListView.swift

```diff
@@ -3,15 +3,19 @@
 import UIKit
 
 final class ListView: UIView {
-  private let listViewCellIdentifier = "ListViewCellIdentifier"
+  private let listViewCellIdentifier = "RepositoryCellView"
```

Comment:
I'd move the identifier to the cell class. We can create a static `reuseIdentifier` in the cell that returns the identifier.
In this way, the ViewController doesn't have to know the exact value for the identifier and, when we work with more complex tables which use different cells, we don't have a long list of identifiers at the beginning of the file.